### PR TITLE
major overhaul of home pages

### LINKED ISSUE #1599: Update home pages
TODO:
- [x] Implement layouts and panels
  - [x] catalog
  - [x] articles
- [x] Update text placeholders
- [x] "You are here" selection
- [x] Settings (see #1673)
- [x] Search tip rotation (see #1674)
- [ ] Update link placeholders (see #1676)

### PR DESCRIPTION
**Pull request by drh-stanford**
This PR fixes #1599.

@jvine how do these screenshots look?

### Catalog home page
screen shot 2017-08-31 at 10 15 32 am

### Article home page

screen shot 2017-08-30 at 4 22 15 pm

### Article search settings ("coming soon")

screen shot 2017-08-31 at 10 34 28 am

### CODE REVIEW

**Comment by jvine** [comment]
Commented on original ticket as well -
@drh-stanford - re catalog home page

- missing the first h1
- it looks like the facets column width is 3 instead of 4...? shouldn't change from current.

File: app/assets/stylesheets/modules/home-page.scss [hunk 1]
```diff
@@ -1,65 +1,91 @@
...
+    padding: 2px 10px;
   }
-  .features-body {
-    padding-bottom: 100px;
+
+  .home-page-you-are-here {
+    .panel .panel-body {
+      -webkit-box-shadow: 0px 0px 10px 0px rgba(233,131,0,1);
+      -moz-box-shadow: 0px 0px 10px 0px rgba(233,131,0,1);
+      box-shadow: 0px 0px 10px 0px rgba(233,131,0,1);
```
Comment (jkeck) [inline app/assets/stylesheets/modules/home-page.scss]: I _think_ we can get away w/ just `box-shadow` here given that we have `autoprefixer-rails`.

File: app/assets/images/homepage/search.svg [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+<!-- Generator: Adobe Illustrator 16.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Filled_Icons" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
+	 y="0px" width="24px" height="24px" viewBox="0 0 24 24" enable-background="new 0 0 24 24" xml:space="preserve">
+<g>
+	<path d="M9,18c2.131,0,4.089-0.749,5.633-1.992l7.658,7.697c0.389,0.392,1.021,0.394,1.414,0.003
+		c0.392-0.39,0.393-1.023,0.004-1.414l-7.668-7.707C17.264,13.052,18,11.111,18,9c0-4.963-4.037-9-9-9S0,4.037,0,9
+		C0,13.962,4.037,18,9,18z M9,2c3.859,0,7,3.14,7,7c0,3.859-3.141,7-7,7c-3.86,0-7-3.141-7-7C2,5.14,5.14,2,9,2z"/>
+</g>
+</svg>
```
[the remark this answers is not in the document]
Comment (drh-stanford) [reply app/assets/images/homepage/search.svg]: it looked different enough to me that i kept it as-is. i'll defer to @jvine on whether we can use the glyphicon-search instead

File: app/views/articles/_home_page.html.erb [hunk 3]
```diff
@@ -1,39 +1,62 @@
...
-          <div class="external-link-icon"/>
+      </div>
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <h2>Search tip</h2>
         </div>
-      </li>
-    </ul>
+        <div class="panel-body">
+          <% search_tip_n = (1..t('searchworks.articles.search_tips.max').to_i).to_a.sample %>
```
[the remark this answers is not in the document]
Comment (drh-stanford) [reply app/views/articles/_home_page.html.erb]: i only have the one search tip right now but i wanted to have something there to do a random rotation. i think what i'll do is remove the rotation code and break this feature into another ticket.

File: app/assets/stylesheets/modules/home-page.scss (hunk 1 above)
Comment (drh-stanford) [reply app/assets/stylesheets/modules/home-page.scss]: yeah i'm worried about changing this because i don't know how to test its removal 🤷‍♂️

Comment (jkeck) [reply app/assets/stylesheets/modules/home-page.scss]: Yeah...actually, I just pulled those prefixes and checked the generated scss and they were not there as I expected.  :shipit:

File: app/assets/images/homepage/search.svg (hunk 2 above)
[the remark this answers is not in the document]
Comment (jvine) [reply app/assets/images/homepage/search.svg]: @drh-stanford @jkeck The svg matches the website, and is the icon in http://embed-dev.stanford.edu/styleguide/icons (sul-i-search f12d). It does not match the search icon we're using in SearchWorks, which I think is the bootstrap default. I'd forgotten we have the sul icons in SW for the Resource types, duh - if we can use both (home page and search box) from that source, that would be fine.

**Pull request merged**